src/@types/fluro.d.ts

```diff
@@ -24,6 +24,7 @@ declare module 'fluro' {
     verified: boolean
     account: Account
     permissionSets: { [key: string]: PermissionSet }
+    contacts: string[]
```

Comment:
Added field here for now, after discussion with Tatai - User will be moved to the useAuth file in a later refactor.